StyleCop.Analyzers/StyleCop.Analyzers/Settings/stylecop.schema.json

```diff
@@ -90,6 +90,16 @@
                 "preserve"
               ],
               "default": "insideNamespace"
+            },
+            "useBlankLinesBetweenUsingGroups": {
```

Comment:
:thought_balloon: It seems like the allowed values would fit the key better if it started with `blank` instead of `useBlank`.